packages/uniforms-antd/src/RadioField.tsx

```diff
@@ -4,14 +4,33 @@ import { FieldProps, connectField, filterDOMProps } from 'uniforms';
 
 import wrapField from './wrapField';
 
+const base64: (string: string) => string =
+  typeof btoa === 'undefined'
+    ? /* istanbul ignore next */ x => Buffer.from(x).toString('base64')
+    : btoa;
+const escape = (x: string) => base64(encodeURIComponent(x)).replace(/=+$/, '');
+
 export type RadioFieldProps = FieldProps<
   string,
   RadioProps,
   { allowedValues?: string[]; transform?: (value: string) => string }
 >;
 
+type ObjectType = {
+  [key: string | number | symbol]: unknown;
+};
+
 const radioStyle = { display: 'block' };
 
+// helper function needed because antd radio.group does not support data-attributes https://github.com/ant-design/ant-design/issues/8561
+const filteredDataAttributes = (props: ObjectType) =>
+  Object.keys(props)
+    .filter(key => key.startsWith('data-'))
+    .reduce((newProps: ObjectType, key: string) => {
+      newProps[key] = props[key];
+      return newProps;
+    }, {});
```

Comment:
I gave it a deeper thought, and I think we should rather make the `data-*` tests optional. I don't think it's good that one theme would render them on the wrapper (thus only once) and some on every item (multiple times).